src/state-summary/state-card-hvac.html

```diff
@@ -0,0 +1,41 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<link rel="import" href="../components/state-info.html">
+
+<dom-module id="state-card-hvac">
+  <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
+  <style>
+  :host {
+    @apply(--paper-font-body1);
+    line-height: 1.5;
+  }
+
+  .state {
+    margin-left: 16px;
+    text-align: right;
+  }
+
+  .target {
+    color: var(--primary-text-color);
+  }
+
+  .current {
+    color: var(--secondary-text-color);
+  }
+  </style>
+  <template>
+    <div class='horizontal justified layout'>
+      <state-info state-obj="[[stateObj]]" in-dialog='[[inDialog]]'></state-info>
+      <div class='state'>
+        <div class='target'>[[computeTargetTemperature(stateObj)]]</div>
+
+        <div class='current'>
+          <span>Currently: </span>
+          <span>[[stateObj.attributes.current_temperature]]</span>
```

Comment:
You are rendering the same in this card as the default state card content so there is no need to create a custom state card.